src/net.cpp

```diff
@@ -2445,8 +2445,11 @@ CNode::~CNode()
 
 void CNode::AskFor(const CInv& inv)
 {
-    if (mapAskFor.size() > MAPASKFOR_MAX_SZ || setAskFor.size() > SETASKFOR_MAX_SZ)
+    if (mapAskFor.size() > MAPASKFOR_MAX_SZ || setAskFor.size() > SETASKFOR_MAX_SZ) {
+        LogPrintf("CNode::AskFor -- Buffers full: mapAskFor.size = %d, setAskFor.size = %d, MAPASKFOR_MAX_SZ = %d, SETASKFOR_MAX_SZ = %d\n",
```

Comment:
"Buffers full" is not quite accurate/descriptive imo, maybe "max size exceeded" instead?
Also would be helpful to see which peer this happened to, add ` peer=%d` at the end of the string (_and `, id` to LogPrintf params ofc_ :) ) pls.